src/Provider/ConcreteProvider.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Yiisoft\EventDispatcher\Provider;
+
+use Psr\EventDispatcher\ListenerProviderInterface;
+
+use function array_values;
+use function class_implements;
+use function class_parents;
+use function get_class;
+
+/**
+ * Provider is a listener provider that registers event listeners for interface names specified explicitly
```

Comment:
You used `Provider` here, probably a c/c from the other class. 😊

Reply:
Fixed.